client/header/activity-panel/activity-card/style.scss

```diff
@@ -113,6 +113,15 @@
 
 	a.components-button.is-button {
 		color: $gray-text;
+		border-color: #ccc;
+		background: #f7f7f7;
```

Comment:
Are there variables for this color?

Reply:
There was for some of these, but not for them all. I've added some that match up with colors inside Gutenberg, just to keep these straight.